ERROR - 2025-10-12 01:09:29 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lms_villanueva', 3306, '', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2025-10-12 01:15:03 --> mysqli_sql_exception: Table 'lms_villanueva.announcements' doesn't exist in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `an...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `an...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `an...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `an...', NULL, false)
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Seeds\AnnouncementSeeder.php(24): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Seeder.php(149): App\Database\Seeds\AnnouncementSeeder->run()
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-12 01:17:37 --> mysqli_sql_exception: Can't DROP FOREIGN KEY `submissions_quiz_id_foreign`; check that it exists in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `su...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `su...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `su...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Forge.php(539): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `su...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Migrations\2025-08-15-124454_CreateSubmissionsTable.php(37): CodeIgniter\Database\Forge->dropForeignKey('submissions', 'submissions_qui...')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-10-12 01:39:34 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `a...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `a...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `a...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `a...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Migrations\2025-10-12-013735_CreateAnnouncementsTable.php(33): CodeIgniter\Database\Forge->createTable('announcements')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateAnnouncementsTable->up()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-12 01:42:09 --> mysqli_sql_exception: Can't DROP FOREIGN KEY `submissions_quiz_id_foreign`; check that it exists in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `su...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `su...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `su...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Forge.php(539): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `su...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Migrations\2025-08-15-124454_CreateSubmissionsTable.php(37): CodeIgniter\Database\Forge->dropForeignKey('submissions', 'submissions_qui...')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-12 01:43:47 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `a...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `a...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `a...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `a...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Migrations\2025-10-12-013735_CreateAnnouncementsTable.php(33): CodeIgniter\Database\Forge->createTable('announcements')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateAnnouncementsTable->up()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-12 01:45:16 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `a...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `a...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `a...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `a...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Migrations\2025-10-12-013735_CreateAnnouncementsTable.php(33): CodeIgniter\Database\Forge->createTable('announcements')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateAnnouncementsTable->up()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-12 01:48:37 --> mysqli_sql_exception: Can't DROP FOREIGN KEY `submissions_quiz_id_foreign`; check that it exists in C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `su...', 0)
#1 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `su...')
#2 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `su...')
#3 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\Forge.php(539): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `su...')
#4 C:\xampp\htdocs\ITE311-VILLANUEVA\app\Database\Migrations\2025-08-15-124454_CreateSubmissionsTable.php(37): CodeIgniter\Database\Forge->dropForeignKey('submissions', 'submissions_qui...')
#5 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-VILLANUEVA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-VILLANUEVA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-VILLANUEVA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-10-12 02:33:14 --> Error: Cannot access protected property CodeIgniter\HTTP\IncomingRequest::$uri
[Method: GET, Route: admin/dashboard]
in APPPATH\Filters\RoleAuth.php on line 21.
 1 SYSTEMPATH\Filters\Filters.php(241): App\Filters\RoleAuth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 2 SYSTEMPATH\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore([...])
 3 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('admin/dashboard', 'before')
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(60): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 02:33:34 --> Error: Cannot access protected property CodeIgniter\HTTP\IncomingRequest::$uri
[Method: GET, Route: teacher/dashboard]
in APPPATH\Filters\RoleAuth.php on line 21.
 1 SYSTEMPATH\Filters\Filters.php(241): App\Filters\RoleAuth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 2 SYSTEMPATH\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore([...])
 3 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('teacher/dashboard', 'before')
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(60): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 02:33:45 --> Error: Cannot access protected property CodeIgniter\HTTP\IncomingRequest::$uri
[Method: GET, Route: student/dashboard]
in APPPATH\Filters\RoleAuth.php on line 21.
 1 SYSTEMPATH\Filters\Filters.php(241): App\Filters\RoleAuth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 2 SYSTEMPATH\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore([...])
 3 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('student/dashboard', 'before')
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(60): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-12 03:04:42 --> ErrorException: filesize(): stat failed for C:\xampp\htdocs\ITE311-VILLANUEVA\writable\session/ci_session87c62c5494a23757b08cecd4cc0070b7
[Method: GET, Route: logout]
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 162.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\ITE311-VILLANUEVA\\writable\\session/ci_session87c62c5494a23757b08cecd4cc0070b7', 'C:\\xampp\\htdocs\\ITE311-VILLANUEVA\\system\\Session\\Handlers\\FileHandler.php', 162)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(162): filesize('C:\\xampp\\htdocs\\ITE311-VILLANUEVA\\writable\\session/ci_session87c62c5494a23757b08cecd4cc0070b7')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('87c62c5494a23757b08cecd4cc0070b7')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(243): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(715): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::session(Object(Config\Session), false)
 8 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(668): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Config\BaseService.php(202): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 SYSTEMPATH\Common.php(995): CodeIgniter\Config\BaseService::get('session')
13 SYSTEMPATH\Common.php(968): service('session')
14 APPPATH\Controllers\Auth.php(97): session()
15 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->logout()
16 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
17 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
20 FCPATH\index.php(60): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
